### ISSUE TITLE
Release Azure.ResourceManager.GraphServices first beta

### ISSUE THREAD

**Issue by ArthurMa1978** [Azure/azure-sdk-for-net#33730]
No Later than 2023-02-24

Board review approved: https://github.com/Azure/azure-sdk-pr/issues/847

**Comment by ArcturusZhang** [comment]
I did not find the corresponding swagger for this RP. The swagger is in the private repo now and not merged into the public swagger repo

**Comment by ArthurMa1978** [comment]
Pending on service team upgrade the swagger spec
